### DISCUSSION TITLE
Pear PHP

### DISCUSSION THREAD

**Discussion by PacerEdition** [rectorphp/rector#8731] [Q&A]
Is there any support within Rector to update Pear files from [Pear website](https://pear.php.net/) from php5.4 to current PHP version?

`due to: 
         "Syntax error, unexpected T_NEW, Syntax error, unexpected T_NEW". On line: 94`

**Comment by TomasVotruba** [comment]
This might help: https://getrector.com/documentation/how-to-run-on-php-53
